lib/tram/page/section.rb

```diff
@@ -9,11 +9,10 @@ class Tram::Page
   class Section
     extend Dry::Initializer
     param  :name,   proc(&:to_sym)
-    option :method, proc(&:to_s),    default: -> { name }, as: :method_name
-    option :value,  proc(&:to_proc), optional: true,       as: :block
-    option :if,     proc(&:to_s),    optional: true,       as: :positive
-    option :unless, proc(&:to_s),    optional: true,       as: :negative
-    option :skip,   true.method(:&), optional: true
+    option :method, proc(&:to_s),    default: -> { name }
```

Comment:
I think we'd better rename these methods because the original ones can provide problems during debugging
(but I agree to not renaming `value` to `block`)

Reply:
Not to mention using `attributes` adds a bunch of excessive steps in a runtime